src/app/(auth)/login/page.tsx

```diff
@@ -44,7 +57,13 @@ export default function Login() {
     } else {
       profile?.loadProfile();
       setErrorMessage('');
-      push('/cases');
+      // conditional routing after log in
+      if (!profile?.profileData) push(CONFIG.onboardingHome);
+      else if (profile.roles.map(r => r.role).includes('ATTORNEY'))
+        push('/cases');
```

Comment:
lowkey could be good to add `CONFIG` values for these as well, just so it's easier to refactor in case the NPO ever decides to change the page name

Reply:
sounds good! i add cases, lca and language support routes to CONFIG. Did we want to add routes for all other pages too (e.g. login, signup)? I guess those are pretty standard, but just for consistency's sake? 
If so, I could add the login route since i'm using in this PR

Reply:
ye feel free to add them!